Lib/test/test_sysconfig.py

```diff
@@ -472,9 +472,10 @@ def test_srcdir(self):
             # should be a full source checkout.
             Python_h = os.path.join(srcdir, 'Include', 'Python.h')
             self.assertTrue(os.path.exists(Python_h), Python_h)
-            # <srcdir>/PC/pyconfig.h always exists even if unused on POSIX.
-            pyconfig_h = os.path.join(srcdir, 'PC', 'pyconfig.h')
-            self.assertTrue(os.path.exists(pyconfig_h), pyconfig_h)
+            if os.name == 'nt':
+                # <srcdir>/PC/pyconfig.h only exists on Windows.
```

Comment:
Honestly, it's probably a better long-term move to get rid of `PC/pyconfig.h` completely and generate it in `Include` where the POSIX one would be.

Eventually (though I suspect it's "likely never") we'll get a combined build system. At that point, it'll be much more sensible to have a coherent set of include files.

The code that tries to include header files in installers already assumes it can pick it up directly from a repository checkout (`Tools/msi/dev` and `PC/layout`, IIRC). That will need additional logic to pick it up from somewhere else - I'd suggest the build directory `$(Py_OutDir)` along with every other generated file that we include in the distribution.